src/CMakeLists.txt

```diff
@@ -22,6 +22,14 @@
 # THE SOFTWARE.
 #####################################################################################
 
+cmake_policy(SET CMP0057 NEW)
+
+include(GenerateExportHeader)
+
+if(NOT HIP_PATH)
+    set(HIP_PATH "$ENV{HIP_PATH}")
```

Comment:
`CMAKE_PREFIX_PATH` should be used to tell cmake where hip is. Dont use custom cmake variables for dependencies.

Reply:
Fixed on `uif2-initial` branch - not yet pushed to repo.

Reply:
Currently Windows installer use HIP_PATH and ROCM_PATH to locate HIP/ROCm installation. We will keep it.